tests/unit/UrlTest.js

```diff
@@ -1,221 +1,6 @@
 const Url = require('../../src/libs/Url');
 
 describe('Url', () => {
-    describe('getURLObject()', () => {
```

Comment:
All these example urls should still be unit tested in `getPathFromURL`.

Reply:
Added tests for getPathFromURL